src/solo3v3_sc.h

```diff
@@ -66,7 +66,6 @@ class Solo3v3BG : public AllBattlegroundScript
     uint32 oldTeamRatingHorde;
 
     void OnQueueUpdate(BattlegroundQueue* queue, uint32 /*diff*/, BattlegroundTypeId bgTypeId, BattlegroundBracketId bracket_id, uint8 arenaType, bool isRated, uint32 /*arenaRatedTeamId*/) override;
-    void OnBattlegroundUpdate(Battleground* bg, uint32 /*diff*/) override;
```

Comment:
why did you delete this?

Reply:
We have the hook OnArenaStart now, so we can use it to call CheckStartSolo3v3.

But we could use OnBGUpdate and check if there's a player that didn't accept the queue and then end the solo earlier, while not in progress, instead of waiting the arena to start